cloudwash/providers/aws.py

```diff
@@ -60,13 +60,34 @@ def dry_pips():
                     [dry_data["PIPS"]["delete"].append(dpip["AllocationId"]) for dpip in rpips]
                 return rpips
 
+            def dry_stacks():
+                rstacks = []
+                if settings.aws.criteria.stacks.unassigned:
+                    [
+                        rstacks.append(stack.name)
+                        for stack in aws_client.list_stacks()
+                        if (
+                            total_running_time(stack).minutes
+                            >= settings.aws.criteria.stacks.sla_minutes
+                            or stack.name.startswith(settings.aws.criteria.stacks.delete_stack)
+                        )
+                        and stack.name not in settings.aws.exceptions.stacks.stack_list
+                    ]
+                    [dry_data['STACKS']['delete'].append(stack) for stack in rstacks]
+
+                return rstacks
+
             # Remove / Stop VMs
             def remove_vms(avms):
                 # Remove VMs
                 [aws_client.get_vm(vm_name).delete() for vm_name in avms["delete"]]
                 # Stop VMs
                 [aws_client.get_vm(vm_name).stop() for vm_name in avms["stop"]]
 
+            # Delete CloudFormations
+            def remove_stacks(stacks):
+                [aws_client.get_stack(stack).delete() for stack in stacks]
```

Comment:
```suggestion
                [aws_client.get_stack(stack_name).delete() for stack_name in stacks]
```
Not a blocker but for consistency!